docs/CRSF.md

```diff
@@ -14,7 +14,10 @@ Optional but recommended steps:
 - Set the receiver into "MAVLink mode" (described below).
 - Install the Yaapu app (described below).
 
-The mLRS default parameter settings follow the below recommendations. That is, with a freshly flashed mLRS Tx module and receiver all you need to do is to set up your radio and flight controller!
+> [!TIP]
+> The mLRS default parameter settings follow the below recommendations. That is, with a freshly flashed mLRS Tx module and receiver all you need to do is to set up your radio and flight controller.
+
+It is recommeneded that for first setup the Tx module and receiver are left in their default configuration. Once your system is up and running there is still time to explore the options for optimizing the performacne.
```

Comment:
Perhaps 'It is recommended for the initial setup the Tx module and receiver are left in their default configuration. Once your system is up and running then you can explore the options for further optimizing performance.'

Reply:
done